config-default.yml

```diff
@@ -522,7 +522,7 @@ voice_gate:
     minimum_messages: 50  # How many messages a user must have to be eligible for voice
     bot_message_delete_delay: 10  # Seconds before deleting bot's response in Voice Gate
     minimum_activity_blocks: 3  # Number of 10 minute blocks during which a user must have been active
-
+    voice_ping_delete_delay: 60 # Seconds before deleting the bot's ping to user in Voice Gate
```

Comment:
Rest of the config file uses two spaces before the hash.

Reply:
Corrected in latest commit